bindings/nodejs/lib/gpio.js

```diff
@@ -30,12 +30,14 @@ exports.open = function( init ) {
         var direction = init.direction ? init.direction : "out";
         var edge = init.edge ? init.edge : "any";
         var pull = init.pull ? init.pull : "none";
+        var active_low = init.activeLow ? init.activeLow : false;
+        var poll = init.poll ? init.poll : 1000;
```

Comment:
@edersondisouza First of all, I would like to thank you for fixing these and adding sample for JS APIs. 

The default poll value should be 0 as per the spec. 
https://github.com/zolkis/soletta/blob/master/doc/js-spec/gpio.md
`unsigned long long poll = 0;  // no polling`